go/framework/bertybridge/config.go

```diff
@@ -30,6 +30,9 @@ func (c *Config) AppendCLIArg(arg string)                         { c.cliArgs =
 func (c *Config) manager() (*initutil.Manager, func(), error) {
 	manager := initutil.Manager{}
 
+	// FIXME: Disable this option once the discovery is fixed.
+	manager.Node.Protocol.RelayHack = true
```

Comment:
take the habit to update the js code, else we will have some rules here, some other in the other place